server/views/partials/expectedArrivalCardMacro.njk

```diff
@@ -11,16 +11,15 @@
             </aside>
             <div class="app-card__content">
                 <header class="app-card__header govuk-!-margin-bottom-4">
-                    <h2 class="app-card__title" data-qa="{{moveType}}-title-{{loopIndex}}">           
-                        {% if user.isReceptionUser and 
-                            (
-                                ['COURT', 'CUSTODY_SUITE'].includes(arrival.fromLocationType) 
-                                and not arrival.isCurrentPrisoner
-                            ) 
-                        %}
-                            <a href="/prisoners/{{ arrival.id }}/confirm-arrival" class="govuk-link govuk-link--no-visited-state" data-qa="confirm-arrival"> {{ arrival.lastName }}, {{ arrival.firstName }}</a>
-                        {% else %}
-                            {{ arrival.lastName }}, {{ arrival.firstName }}
+                    <h2 class="app-card__title" data-qa="{{moveType}}-title-{{loopIndex}}">     
+                        {% if user.isReceptionUser %}
+                            {% if arrival.fromLocationType === 'PRISON' %}
+                                    <a href="/prisoners/{{ arrival.prisonNumber }}/check-transfer" class="govuk-link govuk-link--no-visited-state" data-qa="confirm-arrival"> {{ arrival.lastName }}, {{ arrival.firstName }}</a> 
+                            {% elif not arrival.isCurrentPrisoner %}
+                                    <a href="/prisoners/{{ arrival.id }}/confirm-arrival" class="govuk-link govuk-link--no-visited-state" data-qa="confirm-arrival"> {{ arrival.lastName }}, {{ arrival.firstName }}</a>
+                            {% else  %}
+                                    {{ arrival.lastName }}, {{ arrival.firstName }}
+                            {%  endif %}
                         {%  endif %}
```

Comment:
PCS and court

if (user.isReceptionUser and not arrival.isCurrentPrisoner) {
show link 
} else name

transfer:

if (user.isReceptionUser) {
  show link 
} else  {
  name
}